### PR TITLE
sea70 for QUICCHE

### PR DESCRIPTION
**Pull request by callumrollo**

### CODE REVIEW

**Comment by callumrollo** [comment]
@MOchiara can you review this please? This one has a lot of changes and is for an external project (QUICCHE) so there's a high chance of errors

File: mission_yaml/SEA70_M27.yml [hunk 1]
```diff
@@ -0,0 +1,627 @@
...
+  references:     "created with pyglider https://github.com/c-proof/pyglider"
+  source:     Observational data from a profiling glider.
+  standard_name_vocabulary: CF STandard Name Table v49
+  transmission_system: IRIDIUM
+  glider_name: Scratch
+  glider_serial: "70"
+  wmo_id: "8901002"
+  sea_name: "South Atlantic"
+  comment: 'Calibration date for OCR504 uncertain'
+  contributor_name: Callum Rollo, Louise Biddle, Olle Petersson, Aleksandra Mazur, Marcus Melin
```
Comment (MOchiara) [inline mission_yaml/SEA70_M27.yml:33]: Should this be modified given this is not a VOTO glider? Like Bastien name and Estel etc

File: mission_yaml/SEA70_M27.yml [hunk 2]
```diff
@@ -0,0 +1,627 @@
...
+    long_name: SeaBird OCR504
+    make_model: SeaBird OCR504
+    factory_calibrated: "Yes"
+    calibration_date: "2021-03-08"
+
+
+# map between glider variables and netcdf variables.  This shouldn't
+# change too much.
+netcdf_variables:
+  keep_variables: ['temperature', 'oxygen_concentration', 'chlorophyll', 'nitrate_concentration', 'down_irradiance_380']
```
Comment (MOchiara) [inline mission_yaml/SEA70_M27.yml:92]: Shouldn't this be 'downwelling_PAR'

Comment (callumrollo) [reply mission_yaml/SEA70_M27.yml:92]: As long as it's a variable from the OCR it's fine. Likewise conductivity would work in place of temperature

File: mission_yaml/SEA70_M27.yml [hunk 3]
```diff
@@ -0,0 +1,627 @@
...
+
+  time_uv:
+    comment:       Not computed
+    long_name:     Time
+    standard_name: time
+    calendar:      gregorian
+    units:         seconds since 1970-01-01T00:00:00Z
+    observation_type: calculated
+
+  instrument_ctd:
```
[the remark this answers is not in the document]
Comment (callumrollo) [reply mission_yaml/SEA70_M27.yml:617]: thanks, corrected

File: mission_yaml/SEA70_M27.yml (hunk 1 above)
Comment (callumrollo) [reply mission_yaml/SEA70_M27.yml:33]: corrected

File: mission_yaml/SEA70_M27.yml [hunk 4]
```diff
@@ -30,8 +30,8 @@ metadata:
   wmo_id: "8901002"
   sea_name: "South Atlantic"
   comment: 'Calibration date for OCR504 uncertain'
-  contributor_name: Callum Rollo, Louise Biddle, Olle Petersson, Aleksandra Mazur, Marcus Melin
-  contributor_role: Data Processor, PI, Head of Operations, Glider Technician, Glider Technician
+  contributor_name: Callum Rollo, Bastien Queste, Estel Font
+  contributor_role: Data Processor, PI, glider techinician
```
Comment (MOchiara) [inline mission_yaml/SEA70_M27.yml:34]: Spelling of technician

**Pull request merged**